### PR TITLE
Mt fixes/v9

### PR DESCRIPTION
**Pull request by victorjulien**
Replace #9371 

SV_BRANCH=pr/1349

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 15568

File: doc/userguide/configuration/multi-tenant.rst [hunk 1]
```diff
@@ -217,6 +218,29 @@ Tenants can be mapped to vlan ids.
 The registration of tenant and tenant handlers can be done on a
 running engine.
 
+Reloads
```
Comment (jasonish) [inline doc/userguide/configuration/multi-tenant.rst:221]: nit: seems like a funny title.. "Reloading" would be better. Can be done later.

**Review by jasonish — APPROVED** [review]
Not as thorough review, or as much testing as I would have liked, but nothing jumped out.  And in testing, nothing adverse in non-MT usage.  Only light MT use as my power cut mid vlan testing.  I plan to put into into multi-device testing for a longer period of time soon.

**Comment by victorjulien** [comment]
Replaced by #9380. @jasonish can you approve that too? Only diff is rebase + added one ticket ref.

**Pull request closed without merging**